system/lib/wasmfs/file_table.cpp

```diff
@@ -11,8 +11,6 @@
 
 namespace wasmfs {
 
-std::vector<std::shared_ptr<OpenFileState>> FileTable::entries;
```

Comment:
@kripken @tlively Updated the PR comment to explain why we needed to delete this.